server/server.go

```diff
@@ -79,6 +79,12 @@ func NewServer(
 // Store or Secrets Manager) and write the secrets to the mount point. The
 // version ids of the secrets are then returned to the driver.
 func (s *CSIDriverProviderServer) Mount(ctx context.Context, req *v1alpha1.MountRequest) (response *v1alpha1.MountResponse, e error) {
+	// Log out the write mode
+	if s.driverWriteSecrets == true {
```

Comment:
It's been a while since I've written go, can't you just
```go
if s.driverWriteSecrets {
...
}
```
I don't think that evaluation is necessary.